src/models/Organelle.tsx

```diff
@@ -33,25 +33,18 @@ export const Organelle: any = types
     }
   }))
   .actions(self => ({
-    incrementSubstance(substanceType: SubstanceType, amount: number) {
+    incrementSubstance(substanceType: SubstanceType, amount: number, currentTime: number) {
       let substanceLevel: ISubstance = self.substanceDeltas.get(substanceType);
-      if (substanceLevel) {
-        substanceLevel.manuallyIncrement(amount, self);
-      } else {
-        self.substanceDeltas.set(substanceType, Substance.create({
```

Comment:
I realized that `substanceDeltas` should always be non-null since they're now set every `step`, so I removed this null check.